### ISSUE TITLE
Allow torch.mean to accept a tuple of dimensions

### ISSUE THREAD

**Issue by carlosgmartin** [pytorch/pytorch#13279]
## 🚀 Feature
`torch.mean` currently only accepts a single dimension. It should also accept a tuple of dimensions like `torch.sum` does.

## Motivation

Makes code cleaner and more concise when taking the mean of some quantity across multiple dimensions. An example of this is taking the mean of some pixel-wise quantity over an image (which has two dimensions, and potentially a color channel).

## Alternatives

Current workaround is chaining `.mean`s repeatedly, as in `array.mean(0).mean(0).mean(0)` to take the mean over the first three dimensions of an array.

**Comment by vishwakftw** [comment]
Seems to be a duplicate of https://github.com/pytorch/pytorch/issues/9703 .

**Comment by soumith** [comment]
closing as duplicate of #9703 but we should start work on that soon (we were blocked on TensorIterator reductions landing, and they've landed now)
